### ISSUE TITLE
Custom two space version of PSR2 doesn't work for mult-line function declarations and calls

### ISSUE THREAD

**Issue by codeinabox** [squizlabs/PHP_CodeSniffer#1636]
I've tried creating a custom two space version of PSR2 as follows

```xml
<?xml version="1.0"?>
<ruleset name="Acme Corp">
    <description>PSR2 with two spaces</description>
    <arg name="tab-width" value="2"/>
    <rule ref="PSR2" />
    <rule ref="Generic.WhiteSpace.ScopeIndent">
        <properties>
            <property name="indent" value="2"/>
            <property name="tabIndent" value="false"/>
        </properties>
    </rule>
</ruleset>
```

However for multi-line function declarations and calls it's expecting 6 space indenting when it should should be 4 as the indent is meant to be 2.

```php
  public function sendOrder(
    $orderTime,
    $user,
    $products,
    $payment,
    $billId,
    $loyalty,
    &$extras,
    OrderType $orderType,
    $itemsType = 'portion'
  ) {
    $request = new Request(
      'POST',
      '/',
      [
        'Content-Type' => 'text/xml'
      ],
      $this->buildPayload(...$products)
    );

    $this->client->send($request);
  }
```

```
----------------------------------------------------------------------------------------------------------
FOUND 24 ERRORS AND 1 WARNING AFFECTING 25 LINES
----------------------------------------------------------------------------------------------------------
 44 | WARNING | [ ] Line exceeds 120 characters; contains 131 characters
 52 | ERROR   | [x] Multi-line function declaration not indented correctly; expected 6 spaces but found 4
 53 | ERROR   | [x] Multi-line function declaration not indented correctly; expected 6 spaces but found 4
 54 | ERROR   | [x] Multi-line function declaration not indented correctly; expected 6 spaces but found 4
 55 | ERROR   | [x] Multi-line function declaration not indented correctly; expected 6 spaces but found 4
 56 | ERROR   | [x] Multi-line function declaration not indented correctly; expected 6 spaces but found 4
 57 | ERROR   | [x] Multi-line function declaration not indented correctly; expected 6 spaces but found 4
 58 | ERROR   | [x] Multi-line function declaration not indented correctly; expected 6 spaces but found 4
 59 | ERROR   | [x] Multi-line function declaration not indented correctly; expected 6 spaces but found 4
 60 | ERROR   | [x] Multi-line function declaration not indented correctly; expected 6 spaces but found 4
 63 | ERROR   | [x] Multi-line function call not indented correctly; expected 8 spaces but found 6
 64 | ERROR   | [x] Multi-line function call not indented correctly; expected 8 spaces but found 6
 65 | ERROR   | [x] Multi-line function call not indented correctly; expected 8 spaces but found 6
 67 | ERROR   | [x] Multi-line function call not indented correctly; expected 8 spaces but found 6
 68 | ERROR   | [x] Multi-line function call not indented correctly; expected 8 spaces but found 6
```

**Comment by gsherwood** [comment]
Unfortunately, PHPCS doesn't yet have shared config settings, so you need to set the indent settings for the other sniffs that are reporting errors as well. Both those sniffs are extending PEAR sniffs, so you need to know where to look. But all indent settings are at least called `indent` so you can always give that a try if you find more.

In your specific case, try this for your ruleset:
```xml
<?xml version="1.0"?>
<ruleset name="Acme Corp">
    <description>PSR2 with two spaces</description>
    <arg name="tab-width" value="2"/>
    <rule ref="PSR2" />
    <rule ref="Generic.WhiteSpace.ScopeIndent">
        <properties>
            <property name="indent" value="2"/>
        </properties>
    </rule>
    <rule ref="Squiz.Functions.MultiLineFunctionDeclaration">
        <properties>
            <property name="indent" value="2" />
        </properties>
    </rule>
    <rule ref="PSR2.Methods.FunctionCallSignature">
        <properties>
            <property name="indent" value="2" />
        </properties>
    </rule>
</ruleset>
```

**Comment by codeinabox** [comment]
Thank you for the quick response. That worked perfectly

**Comment by muuvmuuv** [comment]
Hey @gsherwood,

I'm using your example above and still have 4 spaces in `try` and `if` statements. I'm using VSCode with the Unibeautify extension and the custom config option for phpcs. Do I have to add something else to get this working?

**Comment by muuvmuuv** [comment]
Okay, I found the solution in an extension that enables mixed formatting in VSCode. Sorry :)
